packages/amplication-client/src/Entity/EntityFieldList.tsx

```diff
@@ -75,12 +75,14 @@ const EntityFieldList = React.memo(({ entityId }: Props) => {
 
   return (
     <>
-      <div className={`${CLASS_NAME}__header`}>
-        <SearchField
-          label="search"
-          placeholder="search"
-          onChange={handleSearchChange}
-        />
+      <div className={`${CLASS_NAME}__container`}>
```

Comment:
why do you need two divs ? instead of one ? if you need a container please add it to whole component and not as a wrapper to header